src/test/java/com/squareup/javapoet/AnnotationSpecTest.java

```diff
@@ -104,16 +109,61 @@
   @Rule public final CompilationRule compilation = new CompilationRule();
 
   @Test public void equalsAndHashCode() {
-    AnnotationSpec a = AnnotationSpec.builder(AnnotationC.class).build();
-    AnnotationSpec b = AnnotationSpec.builder(AnnotationC.class).build();
+    AnnotationSpec a = AnnotationSpec.get(AnnotationC.class);
+    AnnotationSpec b = AnnotationSpec.get(AnnotationC.class);
     assertThat(a.equals(b)).isTrue();
     assertThat(a.hashCode()).isEqualTo(b.hashCode());
-    a = AnnotationSpec.builder(AnnotationC.class).addMember("value", "$S", "123").build();
-    b = AnnotationSpec.builder(AnnotationC.class).addMember("value", "$S", "123").build();
+    a = AnnotationSpec.get(AnnotationC.class, "123");
+    b = AnnotationSpec.get(AnnotationC.class, "123");
     assertThat(a.equals(b)).isTrue();
     assertThat(a.hashCode()).isEqualTo(b.hashCode());
   }
 
+  @Test public void createSimpleAnnotation() {
+    String pack = "com.squareup.javapoet.AnnotationSpecTest";
```

Comment:
If there's ever a v4, probably. The whole reason JavaPoet is called JavaPoet (and not JavaWriter v3) was because we needed a new package name to not clash with JavaWriter v2 in classpaths where both are being used!